docs/src/theme/DocPage/styles.module.scss

```diff
@@ -63,7 +67,11 @@
   }
 
   .docPage {
-    grid-template-columns: 298px 1fr;
+    grid-template-columns: var(--doc-sidebar-width) 1fr;
+  }
+
+  .colMainContent {
+    width: calc(100vw - var(--doc-sidebar-width));
```

Comment:
```suggestion
    width: calc(100vw - (var(--doc-sidebar-width) + 10px));
```
this will help remove the scrollbar

Reply:
There is no horizontal scrolling at all
---

<img width="1431" alt="Screenshot 2021-08-03 at 12 01 56 AM" src="https://user-images.githubusercontent.com/6524419/127907642-1c7a87ea-569c-49ef-9c39-849e1306480b.png">

IMO, no need to change